helpers/sqs_helper.py

```diff
@@ -0,0 +1,108 @@
+"""
+Helper module for sqs messages
+"""
+import json
+import os
+import sys
+import time
+import boto3
+from botocore.exceptions import ClientError
+
+# add project root to sys path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers.base_helper import BaseHelper
+from conf import sqs_conf
```

Comment:
This is confusing. Why is the helper needing to import conf? Shouldn't the test be the only one to know about conf and then pass on that knowledge to the generic helpers? I feel like by putting this line in, we have tied the implementation of a generic SQS helper with a specific conf.

Reply:
right, looking at this now, I feel it is not right to have imported conf inside the helpers